src/psyclone/tests/gocean1p0_transformations_test.py

```diff
@@ -1184,3 +1184,98 @@ def test_module_inline_warning_no_change():
     kern_call = schedule.children[0].children[0].children[0]
     inline_trans = KernelModuleInlineTrans()
     _, _ = inline_trans.apply(kern_call, inline=False)
+
+
+def test_loop_swap_correct():
+    ''' Testing correct loop swapping transform. Esp. try first, middle, and
+    last invokes to make sure the inserting of the inner loop happens at
+    the right place.'''
+
+    psy, invoke = get_invoke("test27_loop_swap.f90", 0)
+    invoke = psy.invokes.get("invoke_loop1")
```

Comment:
You assign to `invoke` twice here. If the first one isn't needed then you can use `_` instead which keeps pylint happy.

Reply:
*grin* There is actually a reason for that: get_invoke() is broken - it should return the first invoke (index 0), but doesn't (because internally it uses keys(), but keys() does not guarantee an order). I might commit a proper fix as well (but I did use a _) :)